docs/sources/tempo/operations/manage-advanced-systems/multitenancy.md

```diff
@@ -13,11 +13,16 @@ aliases:
 Tempo is a multi-tenant distributed tracing backend. It supports multi-tenancy through the use
 of a header: `X-Scope-OrgID`.
 
+The `X-Scope-OrgID` header is used to enforce multi-tenancy in Tempo and Grafana Enterprise Traces.
```

Comment:
Are not this sentence and the one above redundant?

Reply:
Good catch.

Reply:
I combined both of the statements into one line.